project/tagging/factories.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+from django.contrib.contenttypes.models import ContentType
+from factory import DjangoModelFactory, Faker, LazyAttribute
+from .models import CustomTag, TaggedItems
+
+
+class CustomTagFactory(DjangoModelFactory):
+    guid = LazyAttribute(lambda obj: uuid.uuid1())
+    name = Faker("word")
```

Comment:
Lets keep this simple and deal with:
* Setting up a load of example data in our init script
* Explicitly testing cases like "हिंदी में जानकारी" and "🐸"

in the tagging tests, as opposed to relying on loading fixtures in the resource tests to do it indirectly.

Reply:
Yup.  Hard agree here.  Much better to test tagging from _**tagging**_ and only briefly touch on it in related endpoints.  (Methinks that we do need to test that the "connection" is made in a trivial way - that I can add **a** tag at the least to a `resource`, `hangout`, or `learning path` via POST or PATCH..but not more than that.).  Everything else should be in the tagging tests.